### PR TITLE
zmap 2.1.1

### PR DESCRIPTION
**Pull request by dadrian**
Update zmap to 2.1.1

### CODE REVIEW

File: Library/Formula/zmap.rb [hunk 1]
```diff
@@ -1,41 +1,44 @@
...
 
     args = std_cmake_args
+    args << "-DENABLE_DEVELOPMENT=OFF"
     args << "-DRESPECT_INSTALL_PREFIX_CONFIG=ON"
-    args << "-DWITH_JSON=ON" if build.with? "json-c"
     args << "-DWITH_REDIS=ON" if build.with? "hiredis"
+    args << "-DWITH_MONGO=ON" if build.with? "mongo-c"
 
-    system "cmake", ".", *args
+    system "cmake", *args, "."
```
Comment (dunn) [inline Library/Formula/zmap.rb]: Ruby 1.8 doesn't support the splat (`*args`) except at the end of the args list.  Can you change this to `system "cmake", ".", *args`? Thanks!

File: Library/Formula/zmap.rb [hunk 2]
```diff
@@ -1,41 +1,44 @@
...
+  sha256 "29627520c81101de01b0213434adb218a9f1210bfd3f2dcfdfc1f975dbce6399"
 
   head "https://github.com/zmap/zmap.git"
 
   bottle do
     cellar :any
-    sha1 "67ee1c3239a4109088a7b5f9d9db4884640c1920" => :mavericks
-    sha1 "3d780dc9a1125d96294d7071b1cdc359ccf4f302" => :mountain_lion
-    sha1 "b0482d7fedc3063f7218997f4ecdcb93043f898a" => :lion
+    sha256 "829eb6548654b367cffb7eefeaf04ed41425e97c4f276043c0a6bba467a2d68e" => :yosemite
```
Comment (DomT4) [inline Library/Formula/zmap.rb]: Please don't add or remove bottle checksums. We'll do that automatically on pull.

File: Library/Formula/zmap.rb [hunk 3]
```diff
@@ -1,41 +1,44 @@
...
   depends_on "gmp"
   depends_on "libdnet"
-  depends_on "json-c" => :optional
+  depends_on "json-c"
   depends_on "hiredis" => :optional
+  depends_on "mongo-c" => :optional
 
   def install
-    inreplace ["conf/zmap.conf", "src/zmap.c", "src/zopt.ggo"], "/etc", etc
+    zmap_etc = "#{etc}/zmap"
```
Comment (DomT4) [inline Library/Formula/zmap.rb]: Why the need for this line? It doesn't seem to be being called anywhere?

File: Library/Formula/zmap.rb (hunk 2 above)
Comment (dadrian) [inline Library/Formula/zmap.rb]: Will do. I wasn't sure what to do based on the documentation.

File: Library/Formula/zmap.rb (hunk 3 above)
Comment (dadrian) [inline Library/Formula/zmap.rb]: This is leftover from debugging, I should have caught this. Will remove.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/6e1fd40523ba552bc3f24aaffde0a233271a989c. Thank you for your contribution to Homebrew @dadrian!

**Pull request closed without merging**